__tests__/app.test.js

```diff
@@ -117,55 +101,65 @@ describe('GET /api/articles', () => {
+describe('GET /api/articles/:article_id/comments', () =>{
+    it('returns a 200 status code and an array of comments for the given article_id', () => {
         return request(app)
-            .get(`/api/articles/3`)
+            .get(`/api/articles/3/comments`)
             .expect(200)
             .then(({ body }) => {
                 expect(typeof body).toBe('object')
-                expect(body.article[0].title).toBe("Eight pug gifs that remind me of mitch")
-                expect(body.article[0].article_id).toBe(3)
-                expect(body.article[0].author).toBe("icellusedkars")
-                expect(body.article[0].body).toBe("some gifs")
-                expect(body.article[0].created_at).toBe('2020-11-03T09:12:00.000Z')
-                expect(body.article[0].votes).toBe(0)
-                expect(body.article[0].article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+                expect(body.comments.length).toBe(2)
+                expect(body.comments).toBeSortedBy('created_at', {
+                    descending: true,
+                })
+               body.comments.forEach((comment)=>{
+                expect(typeof comment.comment_id).toBe('number')
+                expect(typeof comment.votes).toBe('number')
+                expect(typeof comment.created_at).toBe('string')
+                expect(typeof comment.author).toBe('string')
+                expect(typeof comment.body).toBe('string')
+                expect(typeof comment.article_id).toBe('number')
+               })
+              
+
             })
-           
-          })
-          test('GET:404 sends an appropriate status and error message when given a valid but non-existent id', () => {
-            return request(app)
-              .get('/api/articles/1111')
-              .expect(404)
-              .then(({body}) => {
-                expect(body.msg).toBe('article does not exist');
-              });
-          });
-          test('GET:400 sends an appropriate status and error message when given an invalid id', () => {
-            return request(app)
-              .get('/api/articles/not-an-id')
-              .expect(400)
-              .then((response) => {
+
+    })
+    test('GET:404 sends an appropriate status and error message when given a valid but non-existent id', () => {
+        return request(app)
+            .get('/api/articles/1111/comments')
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe('article_id does not exist');
+            });
+    });
+    test('GET:400 sends an appropriate status and error message when given an invalid id', () => {
+        return request(app)
+            .get('/api/articles/not-an-id/comments')
+            .expect(400)
+            .then((response) => {
                 expect(response.body.msg).toBe('Bad request');
-              });
+            });
```

Comment:
Love these error handling tests! However, you are missing a test here. What if the article does exist, however, has no comments associated with it? For example article_id 13? This should be returning a 200 status with an empty array response